### PR TITLE
Greenify sepa tests

### PR DESCRIPTION
**Pull request by nbessi**

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  system_site_packages: true
+
+before_install:
+  - git clone https://github.com/OCA/partner-contact-management $HOME/partner-contact-management -b 7.0
+  - git clone https://github.com/OCA/bank-statement-reconcile $HOME/bank-statement-reconcile -b 7.0
+  - git clone https://github.com/OCA/account-financial-tools $HOME/account-financial-tools -b 7.0
+  - git clone https://github.com/nbessi/maintainer-quality-tools.git $HOME/maintainer-quality-tools -b nbi-addons_path_missing_on_create
+
+env:
+  - PATH=$HOME/maintainer-quality-tools/travis:$PATH
```
Comment (bwrsandman) [inline .travis.yml]: This is not what `env` should be used for.

You can export `PATH` in the `before_install` section.

`env` should be used to set up different build environments, the same was as it is with the `python` section.
ex:

```
env:
    - ODOO=https://github.com/odoo/odoo`
    - ODOO=https://github.com/OCA/OCB`
```

To run the build twice, once with odoo, once with OCB.

You then use `$ODOO` in the `install` and/or `before_install` sections to have the builds be different in their respective environments.

Comment (nbessi) [inline .travis.yml]: Hello,

Please report your comment here as I used it as model:

https://github.com/gurneyalex/maintainer-quality-tools

Regards

Nicolas

2014-07-03 11:03 GMT+02:00 Sandy notifications@github.com:

> In .travis.yml:
> 
> > +language: python
> > +
> > +python:
> > -  - "2.7"
> >   +
> >   +virtualenv:
> > -  system_site_packages: true
> >   +
> >   +before_install:
> > -  - git clone https://github.com/OCA/partner-contact-management $HOME/partner-contact-management -b 7.0
> > -  - git clone https://github.com/OCA/bank-statement-reconcile $HOME/bank-statement-reconcile -b 7.0
> > -  - git clone https://github.com/OCA/account-financial-tools $HOME/account-financial-tools -b 7.0
> > -  - git clone https://github.com/nbessi/maintainer-quality-tools.git $HOME/maintainer-quality-tools -b nbi-addons_path_missing_on_create
> >   +
> >   +env:
> > -  - PATH=$HOME/maintainer-quality-tools/travis:$PATH
> 
> This is not what env should be used for.
> 
> You can export PATH in the before_install section.
> 
> env should be used to set up different build environments.
> ex:
> - ODOO=https://github.com/odoo/odoo` https://github.com/odoo/odoo
> - ODOO=https://github.com/OCA/OCB` https://github.com/OCA/OCB
> 
> To run the build twice, once with odoo, once with OCB
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/OCA/l10n-switzerland/pull/21/files#r14503594.

## 

Nicolas Bessi
Senior ERP consultant
Business Solution technical manager

Camptocamp SA
PSE A
CH-1015 Lausanne

http://openerp.camptocamp.com

Direct:   +41 21 619 10 26
Office:   +41 21 619 10 10

**Comment by nbessi** [comment]
Travis file must be removed from this PR as it is the one for branch 7.0

**Pull request merged**